### PR TITLE
Add Skills section

### PR DESCRIPTION
**Pull request by vwinnik**

### CODE REVIEW

File: src/home/components/Skills/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  margin: 100px 0px 100px 0px;
+  letter-spacing: 1px;
+  line-height: 60px;
+  text-align: center;
+}
+
+.listSkill {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
```
Comment (hjgonzalez06) [inline src/home/components/Skills/styles.module.scss:14]: Set the `grid-template-columns` property to `repeat(auto-fill, minmax(200px, auto))` so you don't have to set the same property within tablet and mobile styles, as browser will render the appropriate number of columns depending on the screen width.

File: src/home/components/Skills/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+@include for-desktop-up {
+  .title {
+    padding: 0 20px;
+    letter-spacing: 0.6px;
+    line-height: 36px;
+  }
+
+  .listSkill {
+    padding: 0 64px;
```
Comment (hjgonzalez06) [inline src/home/components/Skills/styles.module.scss:52]: I think that set this property here isn't needed, so you could delete it.

File: src/home/components/Skills/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  }
+}
+
+@include for-tablet-down {
+  .listSkill {
+    grid-template-columns: repeat(3, 200px);
```
Comment (hjgonzalez06) [inline src/home/components/Skills/styles.module.scss:58]: If you apply the comment I made above, setting this property here won't be needed.

File: src/home/components/Skills/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+@include for-phone-only {
+  .skills {
+    margin-top: 93px;
+    padding: 56px 0 77px 0;
+  }
+
+  .listSkill {
+    margin: 0;
+    display: block;
```
Comment (hjgonzalez06) [inline src/home/components/Skills/styles.module.scss:84]: If you apply the comment I made above about `grid-template-columns`, setting this property here won't be needed

**Review by nahudalla — APPROVED** [review]

**Pull request merged**